modules/coin/errors.go

```diff
@@ -80,3 +81,11 @@ func ErrNoOutputs() errors.TMError {
 func IsNoOutputsErr(err error) bool {
 	return errors.IsSameError(errNoOutputs, err)
 }
+
+func ErrUnknownKey(mod string) errors.TMError {
```

Comment:
oh awesome I was mostly copying and pasting code from the other files, changing now. cleaner as now need to import `pkg/errors`